tests/e2e/scale/test_scale_osds_fill_75%_reboot_workers.py

```diff
@@ -27,8 +27,12 @@
 @pytest.mark.parametrize(
     argnames=["interface"],
     argvalues=[
-        pytest.param(constants.CEPHBLOCKPOOL),
-        pytest.param(constants.CEPHFILESYSTEM)
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-XXXX")
```

Comment:
Hi @Shrivaibavi , please create polarion test cases (in case you didn't already) and provide them here.
* This is your test case PR so asking you

Reply:
I think we can remove this polarion mark if its going to be added in next PR.

Reply:
Is this being handled in next PR?

Reply:
Yes, not in this PR

Reply:
https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-2192, This is the polarion ID, It would be good if we move this testcase from scale to manage in ocsci, 
WDYT ? If yes i can send a PR to move this testcase and adding polarion ID as well.